plugins/base/src/test/kotlin/translators/DefaultPsiToDocumentableTranslatorTest.kt

```diff
@@ -240,12 +238,17 @@ class DefaultPsiToDocumentableTranslatorTest : BaseAbstractTest() {
             |}
         """.trimMargin(),
             configurationWithNoJVM,
-            pluginOverrides = listOf(OnlyPsiPlugin()) // suppress a descriptor translator because of psi and descriptor translators work in parallel
+            //pluginOverrides = listOf(OnlyPsiPlugin()) // suppress a descriptor translator because of psi and descriptor translators work in parallel
```

Comment:
Is this intentional or a leftover? If intentional, we might have a problem -- this is a recent test from https://github.com/Kotlin/dokka/pull/2400 that checks resolution of ultralight classes specifically, and from what I understand this plugin is needed to avoid conurrency issues

Reply:
This is intentional. Without this, the DPackage does not contain the KotlinSubClass.

Concurrency issues sound problematic--wouldn't these two plugins both run under most conditions?

Reply:
I might be wrong on the concurrency side of things, @vmishenev knows best

Reply:
This test reproduces a fixed  bug #2187 that appeared under specific conditions. If it does not suppress a descriptor translator and is broken, it will be lead to the flaky test.

I suppose you can copy the test for your goals without the plugin `OnlyPsiPlugin`.